Backend/DataAccessLayer/Migrations/201903062125096_initial-create.cs

```diff
@@ -3,10 +3,24 @@ namespace DataAccessLayer.Migrations
     using System;
     using System.Data.Entity.Migrations;
     
-    public partial class createtableuser : DbMigration
+    public partial class initialcreate : DbMigration
     {
         public override void Up()
         {
+            CreateTable(
+                "dbo.Sessions",
+                c => new
+                    {
+                        Id = c.Guid(nullable: false),
+                        Token = c.String(nullable: false),
+                        ExpiresAt = c.DateTime(nullable: false, precision: 7, storeType: "datetime2"),
+                        UpdatedAt = c.DateTime(nullable: false, precision: 7, storeType: "datetime2"),
+                        CreatedAt = c.DateTime(nullable: false, precision: 7, storeType: "datetime2"),
+                        UserId = c.Guid(nullable: false),
+                    })
+                .PrimaryKey(t => t.Id)
+                .ForeignKey("dbo.Users", t => t.UserId, cascadeDelete: true)
+                .Index(t => t.UserId);
```

Comment:
Undo this block of green. It should all be unchanged, including the class name of this file.